fuzzyfinder.go

```diff
@@ -604,16 +604,16 @@ func (f *finder) find(slice interface{}, itemFunc func(i int) string, opts []Opt
 	}
 }
 
-// Find displays a UI that provides fuzzy finding against to the passed slice.
-// The argument slice must be a slice type. If it is not a slice, Find returns
+// Find displays an UI that provides fuzzy finding against the provided slice.
```

Comment:
> an UI 

Should we use `a` instead of `an`?

Reply:
Good catch!
I didn't put much thought to it, but looks like `a` is the way to go here.